Missing `lodash` dep

```
jade-signer-rpc|feat/openrpc-integration-tests⚡ ⇒ open-rpc-test-coverage -s ./openrpc.json --transport=http --reporter=console
internal/modules/cjs/loader.js:628
    throw err;
    ^

Error: Cannot find module 'lodash'
Require stack:
- /usr/local/lib/node_modules/@open-rpc/test-coverage/node_modules/@open-rpc/schema-utils-js/build/src/generate-method-id.js
- /usr/local/lib/node_modules/@open-rpc/test-coverage/node_modules/@open-rpc/schema-utils-js/build/src/index.js
- /usr/local/lib/node_modules/@open-rpc/test-coverage/bin/cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:625:15)
    at Function.Module._load (internal/modules/cjs/loader.js:527:27)
    at Module.require (internal/modules/cjs/loader.js:683:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.<anonymous> (/usr/local/lib/node_modules/@open-rpc/test-coverage/node_modules/@open-rpc/schema-utils-js/build/src/generate-method-id.js:3:16)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:643:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Module.require (internal/modules/cjs/loader.js:683:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/local/lib/node_modules/@open-rpc/test-coverage/node_modules/@open-rpc/schema-utils-js/build/src/generate-method-id.js',
    '/usr/local/lib/node_modules/@open-rpc/test-coverage/node_modules/@open-rpc/schema-utils-js/build/src/index.js',
    '/usr/local/lib/node_modules/@open-rpc/test-coverage/bin/cli.js'
  ]
}
```

--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@open-rpc/meta-schema": "^1.3.1",
-    "@open-rpc/schema-utils-js": "^1.8.0",
+    "@open-rpc/schema-utils-js": "^1.11.4",
     "@types/isomorphic-fetch": "0.0.35",
     "ajv": "^6.10.0",
     "colors": "^1.3.3",

